theano/gpuarray/tests/test_opt.py

```diff
@@ -680,3 +680,18 @@ def randX(*args):
         z = tensor.batched_dot(x, y)
         f = theano.function([x, y], z, mode=mode_with_gpu)
         f(x_val, y_val)
+
+
+def test_crossentropycategorical1hot_lifter():
+    rng = np.random.RandomState(utt.fetch_seed())
+    x = tensor.matrix()
+    y = tensor.lvector()
+    z = tensor.nnet.crossentropy_categorical_1hot(x, y)
+    gx = theano.grad(z.mean(), x)
+    f = theano.function([x, y], [z, gx], mode=mode_with_gpu)
+    theano.printing.debugprint(f, print_type=1)
```

Comment:
remove this line. Otherwise I'm ok to merge. I ran that test in DebugMode and it passed.